hw/dv/tools/dvsim/sim.mk

```diff
@@ -120,10 +120,28 @@ cov_unr_build: gen_sv_flist
 	@echo "[make]: cov_unr_build"
 	cd ${sv_flist_gen_dir} && ${cov_unr_build_cmd} ${cov_unr_build_opts}
 
-cov_unr: cov_unr_build
+cov_unr_vcs: cov_unr_build
 	@echo "[make]: cov_unr"
 	cd ${sv_flist_gen_dir} && ${cov_unr_run_cmd} ${cov_unr_run_opts}
 
+cov_unr_xcelium: 
+ifeq ("$(wildcard $(cov_merge_db_dir))", "")
+	@echo "NO COVERAGE DATABASE DEFINED"
+else
+	@echo "[make]: cov_unr"
+	mkdir -p ${cov_unr_dir}
+	cd ${cov_unr_dir}; ${cov_unr_run_cmd} ${cov_unr_run_opts}
```

Comment:
nit:
```suggestion
	cd ${cov_unr_dir} && ${cov_unr_run_cmd} ${cov_unr_run_opts}
```

Reply:
not sure what the difference is?

Reply:
@weicaiyang's suggestion will explode helpfully if the directory doesn't exist.

Reply:
yeah, if we do `CMD1; CMD2`, when CMD1 fails, it will still execute CMD2. if it's `CMD1 && CMD2`, it will exit when CMD1 fails.

This is more aligned with other targets

Reply:
ahh thank you I was not aware of this!